internal/control_plane/command/start/start.go

```diff
@@ -0,0 +1,31 @@
+package start
+
+import (
+
+	"github.com/spf13/cobra"
+	"octavius/internal/control_plane/server"
+	"octavius/internal/logger"
+)
+
+var createCmd = &cobra.Command{
+	Use:   "start",
+	Short: "Start server",
+	Long:  `Start server at AppPort`,
+	Run: func(cmd *cobra.Command, args []string) {
+		logger.Info("Start Executed")
+		err := server.Start()
+		if err != nil {
+			logger.Panic("Start command err: ", err)
```

Comment:
Do not panic! please use fatal or error instead. and don't capitalize error string.

Reply:
Sure, swapped with fatal as necessary places